test/test_common/simulated_time_system_test.cc

```diff
@@ -68,15 +76,17 @@ class SimulatedTimeSystemTest : public testing::Test {
   SystemTime start_system_time_;
 };
 
-TEST_F(SimulatedTimeSystemTest, AdvanceTimeAsync) {
+INSTANTIATE_TEST_SUITE_P(DelayTimerActivation, SimulatedTimeSystemTest, testing::Bool());
```

Comment:
this might be clearer if you used an enum rather than a bool.

Reply:
What could use more clarity?  This bool is used to set a runtime feature.  Access to the boolean test param is done via a test method with a descriptive name.

Reply:
This is what I was thinking:

```
enum class AdvanceMode { ActivateTImers, DoNotActivateTimers };
```
or similar. I think the helper method where GetParam() is used makes this pretty clear at uses within the test, but I thought putting the enum here would make it more obvious when looking at *this* line why this test is parameterized.

You can probably come up with a better 'opposite' for ActivateTimers than I did, and I'm also fine leaving it as is.

Just a minor suggestion.

Reply:
Done.